src/content/docs/de/guides/migrate-to-astro/from-create-react-app.mdx

```diff
@@ -0,0 +1,369 @@
+
+Aktualisiere alle [Dateiimports](/de/guides/imports/), um die relativen Dateipfade exakt zu referenzieren. Das kannst du mit einem [Importalias](/de/guides/typescript/#import-aliase) erreichen oder indem du den relativen Pfad komplett ausschreibst.
+
+Beachte, dass `.astro` und andere Dateitypen nur mit deren Dateiendung importiert werden können.
+
+```astro title="src/pages/authors/Fred.astro"
+---
+import Card from '../../components/Card.astro';
+---
+<Card />
+```
+
+### CRA Children in Astro
+
+Schreibe jede `{children}`-Instanz in ein Astro `<slot />`. Astro benötigt keine `{children}` als Komponentenattribut und kann automatisch verschaltelte Inhalte in einem `<slot />` rendern.
+
+```astro title="src/components/MyComponent.astro" del={3-9} ins={11-13}
+---
+---
+export default function MyComponent(props) { 
+    return (
+      <div>
+        {props.children}
+      </div>
+    );  
+}
+
+<div>
+  <slot />
+</div>
+```
+
+React-Komponenten, die mehrere `{children}` rendern, können durch [benannte Slots](/de/basics/astro-components/#benannte-slots) in Astro-Komponenten umgewandelt werden.
+
+Lies mehr über [spezifische `<slot />`-Verwendungen in Astro](/de/basics/astro-components/#slots).
+
+### CRA Datenabruf in Astro
+
+Daten abzurufen ist in Astro-Komponenten im Vergleich zu React-Komponenten recht ähnlich, wenn auch mit kleinen Unterschieden.
+
+Um Daten von anderen Dateien in deinem Projekt zu importieren, kannst du alle Instanzen von `useEffect` gegen entweder `Astro.glob()` oder `getCollection()`/`getEntryBySlug()` austauschen.
+
+Um [externe Daten abzurufen](/de/guides/data-fetching/), verwende `fetch()`.
+
+Diese Datenabrufe werden im Frontmatter der Astro-Komponente gemacht und verwenden ein top-level await.
+
+```astro title="src/pages/index.astro"
+---
+import { getCollection } from 'astro:content';
+
+// Hole alle `src/content/blog/` Einträge
+const allBlogPosts = await getCollection('blog');
+
+// Hole alle `src/content/posts/` Einträge
+const allPosts = await Astro.glob('../pages/posts/*.md');
+
+// Rufe externe Daten ab
+const response = await fetch('https://randomuser.me/api/');
+const data = await response.json();
+const randomUser = data.results[0];
+---
+```
+
+Lies mehr über [das Importieren von lokalen Dateien mit `Astro.glob()`](/de/guides/imports/#astroglob), [die Collections API](/de/guides/content-collections/#querying-collections) oder [das Abrufen von externen Daten](/de/guides/data-fetching/).
+
+### CRA Styling in Astro
+
+Eventuell musst du `CSS-in-JS-Bibliotheken` (z.B. styled-components) gegen andere, in Astro verfügbaren CSS-Optionen austauschen.
+
+Falls nötig, ändere etwaige *inline style* Objekte (`style={{ fontWeight: "bold" }}`) zu inline HTML-`style`-Attributen(`style="font-weight:bold;"`). Oder verwenden Astros `<style>`-Tag für gescopte CSS-Styles.
+
+```astro title="src/components/Card.astro" del={1} ins={2}
+<div style={{backgroundColor: `#f4f4f4`, padding: `1em`}}>{message}</div>
```

Comment:
```suggestion
<div style={{backgroundColor: `#f4f4f4`, padding: `1em`}}>{nachricht}</div>
```

Variablennamen kann man mitübersetzen, um zu zeigen, dass sie variabel sind

Reply:
sicher? ich würde davon ausgehen, dass hier eine Variable mit dem Namen `message` gemeint ist

Reply:
ich würde ehrlich gesagt auch in deutscher Dokumentation mit englischen Variablennamen rechnen

Reply:
Ich persönlich finde diese Entscheidung auch seltsam, vielleicht diskutieren wir im Discord im deutschen Thread noch mehr, warum eigentlich diese Regel verfasst wurde:

https://github.com/withastro/docs/blob/main/src%2Fi18n%2Fde%2FREADME.md#L63